src/store/user-context.tsx

```diff
@@ -153,16 +179,26 @@ const UserContextProvider = (props: PropsWithChildren) => {
     });
   };
 
+  const setSortTypeHandler = (sortType: SortTypes) => {
+    dispatchUserAction({
+      type: UserTypes.SET_SORT_TYPE,
+      payload: {
+        sortType,
+      },
+    });
+  };
+
   ///////////////////////////////////////
 
   const contextValue: UserContextObject = {
     cartItems: userState.cartItems,
     address: userState.address,
-    sortType: "",
+    sortType: userState.sortType,
```

Comment:
I'm not sure this belongs here. For now we can leave it as it is. In general, the context should be as specific to a "feature/idea" as possible and shouldn't be polluted with a lot of different things hardly related to each other. We probably don't even need a context for sorting, and even if we do, it might still be a better idea to have a separate context wrapping a specific part of your application that requires that.